lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -63,6 +63,12 @@ def make_complex_mappings
               end
             end
 
+            def map_classification
```

Comment:
Can `classification` this just be included in the array for `make_complex_mappings`? Maybe not because it is only complex for one facility type, but if that is the case I think you should be able to use `complex_mapping` for VHA instead of adding the method `get_mapper`.